src/winforms/toga_winforms/widgets/webview.py

```diff
@@ -23,13 +23,13 @@ def set_content(self, root_url, content):
         self.native.Navigate(Uri(root_url), "_self", None, self.interface.user_agent)
 
     def get_dom(self):
-        self.interface.factory.not_implemented('WebView.get_dom()')
+        return str(self.native.Document)
 
     def set_user_agent(self, value):
         self.native.customUserAgent = value if value else "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10240"
```

Comment:
At column 120: (E501) line too long (189 > 119 characters)

Reply:
The line causing this error was unchanged by me. Do i need to shorten this line? I believe it is line 29.